spock-core/core.gradle

```
@@ -80,10 +80,10 @@ processResources {
   filter(org.apache.tools.ant.filters.ReplaceTokens, tokens: tokens)
 }
 
-task coreConsole(type: JavaExec,
-                 description: 'Start a groovy Console with Spock Core Classpath, useful for AST-Inspection') {
+tasks.register("coreConsole", JavaExec) {
+  description = 'Start a groovy Console with Spock Core Classpath, useful for AST-Inspection'
   main = variant == 2.5 ? "groovy.ui.Console" : "groovy.console.ui.Console"
-  classpath = sourceSets.main.runtimeClasspath + configurations.coreConsoleRuntime
+  classpath(sourceSets.named("runtimeClasspath").map {it.runtimeClasspath }, configurations.named("coreConsoleRuntime"))
```

Is this really correct shouldn't it be: 
```suggestion
  classpath(sourceSets.named("main").map {it.runtimeClasspath }, configurations.named("coreConsoleRuntime"))
```
Incidentally this feels like it is the reverse of the typesafe accessor change.

---

My typo.

---

That feels like it because it is Groovy DSL.
Groovy DSL is longer around than Kotlin DSL.
So accessors there are often eager ones while in Kotlin DSL the accessors are lazy.
While actually for `sourceSets` the change would not strictly be necessary.
The only container really benefiting from the new api currently are task containers.
But it also does not hurt too much to always use the new API and be future-proof. :-)
And if someone invests the time to rewrite the build scripts in Kotlin DSL, it will probably also look more accessor-y again. :-)